packages/replay-next/components/search-files/ResultsList.tsx

```diff
@@ -18,15 +18,17 @@ const EMPTY_ARRAY = [] as SourceSearchResult[];
 export default function ResultsList({
   query,
   streaming,
+  isPending: searchIsPending,
 }: {
   query: string;
   streaming: StreamingSearchValue;
+  isPending: boolean;
 }) {
   const replayClient = useContext(ReplayClientContext);
 
   const { data, status, value: orderedResults = EMPTY_ARRAY } = useStreamingValue(streaming);
 
-  const isPending = status === STATUS_PENDING;
+  const isPending = status === STATUS_PENDING || searchIsPending;
```

Comment:
D'oh this is silly of me. I forgot that we already have an is-pending signal from the Suspense cache. My apologies. We don't need the extra boolean then. Sorry! 🙇🏼